### PR TITLE
Adding debian sample files

### PR DESCRIPTION
**Pull request by Ara4Sh**
Depends on : [docs/421](https://github.com/RackHD/docs/pull/421)
Debian full and minimum sample files added and ubuntu edited to work with new installation graph

### CODE REVIEW

File: example/samples/install_debian_payload_full.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+    "options": {
+        "install-debian": {
```
Comment (lanchongyizu) [inline example/samples/install_debian_payload_full.json]: Replace option `install-debian` with `defaults`?

File: example/samples/install_debian_payload_minimal.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "options": {
+	"install-debian": {
```
Comment (lanchongyizu) [inline example/samples/install_debian_payload_minimal.json]: Same problem?

File: example/samples/install_debian_payload_full.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+                    "mountPoint": "swap",
+                    "size": "8192",
+                    "fsType": "swap"
+                },
+                {
+                    "mountPoint": "/arash",
+                    "size": "10000",
+                    "fsType": "ext4"
+                },
+
```
Comment (lanchongyizu) [inline example/samples/install_debian_payload_full.json]: Please delete the empty line.

File: example/samples/install_debian_payload_full.json [hunk 4]
```diff
@@ -0,0 +1,63 @@
+{
+    "options": {
+        "defaults": {
+            "rootPassword": "RackHDRocks!",
+            "hostname": "rackhd",
+            "domain": "rackhd.io",
+	    "osName": "debian",
+	    "version": "stretch",
+	    "baseUrl": "dists/xenial/main/installer-amd64/current/images/netboot/ubuntu-installer/amd64",
```
Comment (lanchongyizu) [inline example/samples/install_debian_payload_full.json]: Why doese `baseUrl` include `xenial` and `ubuntu-installer`?

File: example/samples/install_debian_payload_full.json [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+	    "repo": "http://172.31.128.1:8080/debian",
+            "users": [
+                {
+                    "name": "arash",
```
Comment (lanchongyizu) [inline example/samples/install_debian_payload_full.json]: Please don't use `arash` as name, instead use `rackhd1`.

File: example/samples/install_debian_payload_full.json [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+                "8.8.8.8",
+                "4.2.2.4"
+            ],
+            "networkDevices": [
+                {
+                    "device": "eno1",
+                    "ipv4": {
+                        "ipAddr": "192.168.1.29",
+                        "gateway": "192.168.1.1",
+                        "netmask": "255.255.255.0"
```
Comment (lanchongyizu) [inline example/samples/install_debian_payload_full.json]: Please add `ipv6` and `vlanIds`. You could look into the ubuntu full payload for an example.

File: example/samples/install_debian_payload_minimal.json [hunk 7]
```diff
@@ -0,0 +1,10 @@
+{
+    "options": {
+	"defaults": {
+		"rootPassword": "SamePassword",
```
Comment (lanchongyizu) [inline example/samples/install_debian_payload_minimal.json]: I think the minimal payload doesn't require `rootPassword`.

File: example/samples/install_debian_payload_full.json [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+                    }
+                }
+            ],
+            "kvm": false,
+            "installDisk": "/dev/sda",
+            "timezone": "Asia/Tehran",
```
Comment (lanchongyizu) [inline example/samples/install_debian_payload_full.json]: Please assign `UTC` to `timezone` as you did in ubuntu full payload.

File: example/samples/install_debian_payload_full.json [hunk 9]
```diff
@@ -0,0 +1,63 @@
+{
+    "options": {
+        "defaults": {
+            "rootPassword": "RackHDRocks!",
+            "hostname": "rackhd",
+            "domain": "rackhd.io",
```
Comment (lanchongyizu) [inline example/samples/install_debian_payload_full.json]: `example.com`?

File: example/samples/install_debian_payload_full.json (hunk 4 above)
Comment (Ara4Sh) [reply example/samples/install_debian_payload_full.json]: ahh !

**Pull request merged**